_sub/compute/eks-addons/dependencies.tf

```diff
@@ -4,42 +4,32 @@
 # CoreDNS:    https://docs.aws.amazon.com/eks/latest/userguide/managing-coredns.html
 # Kube-proxy: https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html
 
+# Get current default build of an add-on (e.g. 'coredns'):
+# aws eks describe-addon-versions --kubernetes-version 1.18 --addon-name coredns | jq -r '.addons[].addonVersions[] | select(.compatibilities[].defaultVersion == true) | .addonVersion'
+
 locals {
   vpccni_version_map = {
-    "1.15" = "1.7.5"
-    "1.16" = "1.7.5"
-    "1.17" = "1.7.5"
-    "1.18" = "1.7.5"
-    "1.19" = "1.7.5"
-    "1.20" = ""
+    "1.18" = "v1.7.5-eksbuild.2"
+    "1.19" = "v1.7.5-eksbuild.2"
+    "1.20" = "v1.7.5-eksbuild.2"
```

Comment:
https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html#updating-vpc-cni-add-on recommends using CNI version 1.7.10

Reply:
It is also CNI version 1.7.10 tested with in the null_provider update